renderer/render_once.sh

```diff
@@ -29,7 +29,7 @@ docker run -e JAVA_TOOL_OPTIONS='-Xmx2g' \
 	-v "$WORKING_DIR/data":/data \
 	-v "$DIR/layers":/layers \
 	ghcr.io/onthegomap/planetiler:latest --area=planet \
-	--download --download-only --only-fetch-wikidata
+	--download --download-only --only-fetch-wikidata --wikidata-max-age=P30D --wikidata-update-limit=100000
```

Comment:
This means that we'll store a timestamp on each wikidata name entry and  each time this runs it will try to re-download up to 100k name entries that are >30 days old.  You could tweak max age based on how often you want to refresh, or lower the update limit to put less load on wikidata query service and trickle-in the updates more gradually over time.